src/pages/home/ReportScreen.js

```diff
@@ -280,35 +282,28 @@ ReportScreen.contextType = DrawerStatusContext;
 ReportScreen.propTypes = propTypes;
 ReportScreen.defaultProps = defaultProps;
 
-export default compose(
+export default compose(withNetwork(), withOnyx({
```

Comment:
Something looks off here and seems we are passing `withWindowDimensions` as a property of the `withOnyx` mapping now.

we probably want something like this:

```js
compose(
    withNetwork(), 
    withWindowDimensions,
    withOnyx({
```

Reply:
migtht have been missed with a merge, adding it